src/workloads/query/TimeseriesTSBSOptimizations.yml

```diff
@@ -0,0 +1,286 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload runs different time-series optimizations with the tsbs dataset. During development, we targeted optimizations
+  that are most relevant to pushdown time-series queries to SBE. Further optimizations should be added to this workload.
+  The data is preloaded by the dsi configuration. See 'configurations/test_control/test_control.tsbs_query.yml' for all 
+  the details. Below are helpful statistics about the collection:
+  1. There are 20736000 documents and 161448 buckets in the collection. 
+  2. "tags" is the metaField and "time" is the timeField.
+  2. The 50th percentile of the number of documents per bucket is 93.
+  4. The 80th percentile of the number of document per bucket is 274.96.
+  3. 40435 buckets are version 1 and 121013 buckets are version 2.
+  4. Predicates of bucket at a time window of 1 hour is expected to hit 0.2% of buckets.
+  5. Predicates on a single measurement value, such as {usage_nice: 5} will hit around 18% of buckets.
```

Comment:
I'd extend this point to say that all measurements vary in [X, Y] range and have similar distributions. I'd also swap points 5 and 6 so that time related points are next to each other.

Reply:
In a different comment thread Gil posted results that show that the distributions aren't actually similar...